db/schema.rb

```diff
@@ -78,6 +78,18 @@
     t.datetime "device_updated_at", null: false
   end
 
+  create_table "prescription_drugs", id: :uuid, default: nil, force: :cascade do |t|
```

Comment:
This can be `id: :uuid` instead?

Reply:
This is true right now for all models/records that are _only_ synced from user. We can change this uniformly for all such models later.